docs/3.0rc/automate/events/webhook-triggers.mdx

```diff
@@ -259,7 +259,7 @@ identifier, so `body.friendly-name` will not work. Use `body['friendly-name']` i
 those cases.
 </Tip>
 
-Prefect Cloud will attempt to parse any other content type (such as `text/plain`) as if it were JSON first.
+Prefect Cloud will attempt to parse any other media type (such as `text/plain`) as if it were JSON first.
```

Comment:
In this case `Content-Type` is the literal name of the field, so we should either update the linter so it doesn't apply here or make it `Content-Type`

Reply:
Done.